src/bci_build/package/base/README.md.j2

```diff
@@ -0,0 +1,30 @@
+# SLE BCI {{ image.os_version.pretty_os_version_no_dash }} Base Container Image
+{% include 'badges.j2' %}
+
+## Description
+
+SUSE Linux Enterprise Base Container Images (SLE BCI) provide truly open,
+flexible, and secure container images and application development tools. The
+images consist of container environments based on SUSE Linux Enterprise and
+designed to be a secure base for any containerized workload.
+
+SLE BCI is freely available, re-distributable, and supported across many
+different environments. These templates and tools address modern, containerized
```

Comment:
which templates and tools? it is not clear here.

Reply:
I'd like to address this in a followup. this one is intended to as closely as possible resemble the existing sle15-image. we can clean it up once this PR has been merged. it is large enough as is.